XamarinFormSample/XamarinFormSample/MainViewModel.cs

```diff
@@ -380,19 +380,8 @@ private BiometricOptions CreateBiometricOptions()
         {
             return new BiometricOptions
             {
-                Android = new BiometricOptionsAndroid
-                {
-                    Title = "Enable biometric title",
-                    Subtitle = "subtitle",
-                    Description = "description",
-                    NegativeButtonText = "Cancel",
-                    AccessConstraint = BiometricAccessConstraintAndroid.BiometricOnly,
-                },
-                Ios = new BiometricOptionsIos
-                {
-                    LocalizedReason = "Reason",
-                    AccessConstraint = BiometricAccessConstraintIos.BiometricAny
-                }
+                Android = new BiometricOptionsAndroid("Enable biometric title", "subtitle", "description", "Cancel", BiometricAccessConstraintAndroid.BiometricOnly, true),
```

Comment:
The intended settings should follow https://github.com/authgear/authgear-sdk-js/blob/master/example/reactnative/src/screens/MainScreen.tsx#L119